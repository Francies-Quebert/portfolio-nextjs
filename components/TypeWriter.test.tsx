import Typewriter from "./TypeWriter";
import { render } from "@testing-library/react";

describe('TypeWriter', () => {
  it('renders correctly', () => {
    const { container } = render(<Typewriter text="hello" delay={10}/>);
    expect(container).toMatchSnapshot();
  });
});