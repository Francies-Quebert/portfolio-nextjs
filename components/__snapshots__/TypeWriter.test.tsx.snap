// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TypeWriter renders correctly 1`] = `
<div>
  <span
    class=""
    style="animation: typing 0.5s steps(40, end);"
  >
    |
  </span>
</div>
`;
